### PR TITLE
Fixes project validation of FileLinks

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

- Fixes [test failure](https://github.com/ITISFoundation/osparc-simcore/pull/2220/checks?check_run_id=2244051004) of ``services/web/server/tests/integration/test_exporter.py::test_import_export_import_duplicate[export_version0]`` caused by a ``ValidationError`` after changes in the constraints of ``BaseFileLink`` models in ``packages/models-library/src/models_library/projects_nodes_io.py``

This issue detected that ``BaseFileLink`` had some inconsistencies when compared with ``api/specs/common/schemas/project-v0.0.1.json``, specifically in the ``dataset`` field. THIS NEEDS review when storage is refactored.

NOTE: This test was temporary disabled in [PR#2220](https://github.com/ITISFoundation/osparc-simcore/pull/2220) so it could be resolved separately for clarity.

## How to test

```cmd
make devenv
source .venv/bin/activate
cd services/web/server
make install-dev
pytest -vv --pdb tests/integration/test_exporter.py
```

### CODE REVIEW

**Review by mguidon — APPROVED** [review]

File: services/web/server/tests/integration/test_exporter.py [hunk 1]
```diff
@@ -191,6 +191,7 @@ async def login_user(client):
 
 
 def get_exported_projects() -> List[Path]:
+    # TODO: explain how to re-generate these files?
```
Comment (GitHK) [inline services/web/server/tests/integration/test_exporter.py:194]: If you unzip them and have a look inside, they are 2 projects with the same structure, containing different versions of the sleeper. The only important thing is the labels of the nodes. They have to follow this pattern:`name_number` like: (sleeper_0, sleeper_1, etc...)
Also these are meant not to be regenerated as they are the "decided" format produced by FormatterV1 and FormatterV2

**Review by GitHK — COMMENTED** [review]
I see that you are still lacking permissions for the sleeper 2.1.1. I had this issue while developing it, but managed to fix it by giving access rights to the service.

File: services/web/server/tests/integration/test_exporter.py (hunk 1 above)
Comment (pcrespov) [reply services/web/server/tests/integration/test_exporter.py:194]: @GitHK I understand that, but do you have a script to generate them?

Comment (GitHK) [reply services/web/server/tests/integration/test_exporter.py:194]: No, and we **should not** have one. They needn't be regenerated. If a new format is released a version3 will be added.

**Review by sanderegg — APPROVED** [review]
pair reviewed. very good!

**Comment by pcrespov** [comment]
> I see that you are still lacking permissions for the sleeper 2.1.1. I had this issue while developing it, but managed to fix it by giving access rights to the service.

where do you see that here? The issue with the permissions was solved in a different PR and I did not touch your "fix" for the tests.

**Comment by GitHK** [comment]
> where do you see that here? The issue with the permissions was solved in a different PR and I did not touch your "fix" for the tests.

In the first failing run before my review of this PR https://github.com/ITISFoundation/osparc-simcore/runs/2275550676#step:9:104

**Comment by pcrespov** [comment]
> > where do you see that here? The issue with the permissions was solved in a different PR and I did not touch your "fix" for the tests.
> 
> In the first failing run before my review of this PR https://github.com/ITISFoundation/osparc-simcore/runs/2275550676#step:9:104

that was something else. It should pass  now.

**Review by GitHK — APPROVED** [review]
👍 reviewed together

**Pull request merged**